development/docker-compose.mysql.yml

```diff
@@ -11,6 +11,7 @@ services:
       - "NAUTOBOT_DB_ENGINE=django_prometheus.db.backends.mysql"
   db:
     image: mysql:8
+    command: --sort-buffer-size=1M
```

Comment:
Probably the wrong place to add this - this is for the local dev environment, not CI.